server/app/core/stt_service.py

```
@@ -0,0 +1,48 @@
+                audio_data,
+                ## language="ja",
+                beam_size=5,
+                word_timestamps=False
+            )
+
+            text = " ".join([segment.text for segment in segments])
+            
+            return text.strip()
+
+        except Exception as e:
+            raise Exception(f"Failed to transcribe audio: {str(e)}")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)